metadata/src/test/java/org/apache/kafka/controller/ClusterControlManagerTest.java

```diff
@@ -272,8 +274,17 @@ public void testRegistrationWithIncorrectClusterId() {
                 (short) 1));
     }
 
+    private static Stream<Arguments> metadataVersions() {
+        return Stream.of(
+                MetadataVersion.IBP_3_3_IV2,
+                MetadataVersion.IBP_3_3_IV3,
+                MetadataVersion.LATEST_PRODUCTION,
```

Comment:
I feel like we should explicitly include `IBP_3_7_IV2` here rather than `LATEST_PRODUCTION`. After all, that is the first JBOD version so it's worth testing specifically, even after `LATEST_PRODUCTION` marches on.

Reply:
Replaced `LATEST_PRODUCTION` with `IBP_3_7_IV2`.